### PR TITLE
Add terminal section

### PR DESCRIPTION
**Pull request by FriedLongJohns**
+ A bunch of other changes I've done and forgot about.

### CODE REVIEW

File: docs/section-8/git.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+## The usuals
+<big> `git fetch` & `git pull` & `git push` </big>
+
+No explanation needed. Fetch before pulling, as usual.
+And do try and remember: Commit before you push, otherwise you're pushing nothing.
+
+## Committing
+<big> `git commit -a -m "Title" -m "Message"` </big>
+Git commit is a bit longer than most of the others, but for good reason. Keep the double-quotes!
```
Comment (ProfessorAtomicManiac) [inline docs/section-8/git.md:47]: maybe explain the -a -m, flags in general

File: docs/section-8/gradle.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Thankfully, there's a file that makes all these things very easy for you.
+
+## The gradlew file
+
+Every EmptyProject20XX repository comes with a `gradlew` file, and by extension every RobotCode20XX will have it as well. Gradlew is an __executable__ file, which means you run it by using the `./` filepath prefix.
+
+Running `./gradlew` without anything else actually just builds the Robot Code. It's that easy - exactly the same thing happens as when you're using VSCode.
+
+!!! note
+	If your terminal tells you `Permission Denied` when running the file, do `sudo chmod 777 ./gradlew`. This gives everyone permission to run the executable.
```
Comment (ProfessorAtomicManiac) [inline docs/section-8/gradle.md:20]: doesn't 'sudo' work only in linux? What about other OS?

**Review by ProfessorAtomicManiac — CHANGES_REQUESTED** [review]
ngl i actually learned some new stuff from that lol

File: docs/section-8/gradle.md (hunk 2 above)
Comment (FriedLongJohns) [reply docs/section-8/gradle.md:20]: we're in bash - it works in all iirc

File: docs/section-8/git.md (hunk 1 above)
Comment (FriedLongJohns) [reply docs/section-8/git.md:47]: Done for -a, -m unnecessary.

File: docs/section-1/version-control.md [hunk 3]
```diff
@@ -13,47 +13,7 @@ VS Code is the text editor we will be using to write and run our code, and shoul
...
-    Mason Cao, 
-    Nick Chen, 
-    Jay Motamarry, 
-    Ylann Bouis,
-    Raya Skaf,
-    Sofie Budman,
-    Tara Krishnan
-
-
-Now stage and commit your changes with the message `add [your name]`, and then push your changes. Refer to the [Git tutorial](https://guides.github.com/introduction/git-handbook/) and [VS Code tutorial](https://code.visualstudio.com/docs/editor/versioncontrol) if needed.
```
Comment (ProfessorAtomicManiac) [inline docs/section-1/version-control.md:57]: See my comment on this in your pr for "Misc Changes". ill approve this branch but u shouldn't make changes on one file in two separate branches

**Review by ProfessorAtomicManiac — CHANGES_REQUESTED** [review]

**Review by ProfessorAtomicManiac — APPROVED** [review]

File: docs/section-1/version-control.md (hunk 3 above)
Comment (FriedLongJohns) [reply docs/section-1/version-control.md:57]: Yeah, I know.
It's not hard to fix the merge conflicts, so I'm not going to worry about it.

**Pull request merged**